.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: test
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: ['windows-latest', 'macos-latest', 'ubuntu-latest']
+        nim: ['devel', 'version-1-4', 'version-1-0']
+
+    name: Nim ${{ matrix.nim }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: alaviss/setup-nim@master
```

Comment:
What's `master` mean here? Nim @ `master` branch? Can we pin to a known version or a commit like we do in the travis file?

Reply:
It means use the master branch of https://github.com/alaviss/setup-nim (which is used to install nim). It doesn't refer to the master branch of Nim.  As written currently, this builds against Nim 1.0.x, Nim 1.4.x and Nim devel.  I'm not sure if alaviss/setup-nim can install a specific commit.

Reply:
Please use the 0.1.0 tag for stability. I'm planning a major overhaul in 0.2.0 which will break things.

Reply:
@alaviss Done. Thanks for the heads up!

Reply:
Can we remove ``devel`` for now then? Having failures because we depend on a moving target is a source of frustration that I don't want to put on myself, or our contributors.

@alaviss I would appreciate if you added support for commits.

Reply:
@dom96 Which commit would you like for now?  It has to be one that has a nightly build.  Unfortunately, the one in travis (8a004e2) doesn't have one.  It looks like that was from April of this year.  If you want one from back then, this looks like a good candidate (though it lacks macOS binaries): https://github.com/nim-lang/nightlies/releases/tag/2020-04-20-devel-1bdc30b

I'd recommend a more recent one, though.  The builds pass on this version: https://github.com/nim-lang/nightlies/releases/tag/2020-11-03-devel-fde17b159fdc96eda5609fd98e1fe3c7a34ef40b

I've hacked together something that will build a specific commit until alaviss/setup-nim supports building from arbitrary (or semi-arbitrary) commits.

Reply:
@dom96 It will be supported in the next version (which will be written in Nim). The current one focus on `fusion` needs, and we don't pin commits there.

Reply:
> which will be written in Nim targeting NodeJS

huh? Why targeting Node?